app/models/physical_ticket.rb

```diff
@@ -6,6 +6,7 @@ class PhysicalTicket < ActiveRecord::Base
   has_many :ticket_scannings
 
   before_create :set_token
+  after_create :send_ticket_confirmation_mail
```

Comment:
there is something I don't like with this. If you but 3 tickets, then we will send 3 different emails. I would instead send this email after creating the physical_tickets in one single email :thinking: